### PR TITLE
Ritai#3

### PR DESCRIPTION
**Pull request by tjh2822**

### CODE REVIEW

File: app/src/view1/view1.html [hunk 1]
```diff
@@ -1 +1 @@
-<p>Hey There Austin!</p>
```
Comment (adg4323) [inline app/src/view1/view1.html:1]: Yo

File: app/src/view1/view1.html [hunk 2]
```diff
@@ -1 +1 @@
...
+<p>Hey There Austin! Tag Me!!!!</p>
```
Comment (mxnavid) [inline app/src/view1/view1.html:1]: Looks alright!

Comment (pdas002) [inline app/src/view1/view1.html:1]: Use proper punctuation next time. Three exclamation points are enough.

**Review by mxnavid — APPROVED** [review]
Approved!

**Review by adg4323 — APPROVED** [review]

**Review by pdas002 — APPROVED** [review]
Good work m8.

**Pull request merged**